tekton-pipeline.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: tekton-instana-version-resource-pipeline
+spec:
+  params:
+    - name: branch
+      type: string
+      default: "main"
+    - name: icr-username
+      type: string
+      default: "iamapikey"
+    - name: icr-password
+      type: string
+      default: ""
+  workspaces:
+    - name: output
+  tasks:
+    - name: git-clone
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+      params:
+        - name: url
+          value: "https://github.com/instana/instana-version-resource.git"
+        - name: revision
+          value: $(params.branch)
+        - name: deleteExisting
+          value: "true"
+
+    - name: build-instana-version-resource-image
+      taskSpec:
+        params:
+          - name: cloned-dir
+            type: string
+            value: instana-version-resource
```

Comment:
I think this should be `default` and not `value`

Reply:
Fixed!